src/frontend/src/views/Data/components/TreeActionMenu/components/TreeTableDownloadMenu/index.tsx

```diff
@@ -1,13 +1,19 @@
 import { Menu, MenuItem, Tooltip } from "czifui";
 import React, { MouseEventHandler, ReactNode, useState } from "react";
+import {
+  AnalyticsTreeDownloadSelectedSamplesTemplate,
+  AnalyticsTreeDownloadTreeFile,
+  EVENT_TYPES,
+} from "src/common/analytics/eventTypes";
+import { analyticsTrackEvent } from "src/common/analytics/methods";
 import { NewTabLink } from "src/common/components/library/NewTabLink";
 import { TREE_STATUS } from "src/common/constants/types";
 import DownloadIcon from "src/common/icons/IconDownloadSmall.svg";
 import { stringGuard } from "src/common/utils";
 import { StyledIcon, StyledIconWrapper } from "../../style";
 
 interface Props {
-  item: TableItem;
+  item: PhyloRun;
```

Comment:
The type change here is safe. This component only gets used in one place, and that place always passes a `PhyloRun`. The more generic `TableItem` left a lot of wiggle room that could possibly break my newly written `analyticsGetData` (and the component itself, it relies on PhyloRun structure implicitly), so this change is to increase type safety if we accidentally change the upstream and forget that this component isn't very generic.